chutney/server/src/main/java/com/chutneytesting/dataset/domain/DatasetService.java

```diff
@@ -45,6 +53,24 @@ public List<DataSet> findAll() {
             .collect(toList());
     }
 
+    @Transactional
```

Comment:
Transactional is necessary here because in the line "Pair.of(t.campaign().title(), ...",  campaign is lazy loaded and needs to be in the same transaction